### ISSUE TITLE
Packages for aarch64 (Raspberry Pi OS, Debian version: 11 (bullseye)) available in the future?

### ISSUE THREAD

**Issue by Krawei** [Betterbird/thunderbird-patches#28]
Dear team,

I noticed your project via linuxnews.de and I wanted to test it under Rapsberry Pi OS 64-Bit. Unfortunately your decent fork isn't available in the debian repository nor as a AARCH64-*.deb via your project's web page (https://www.betterbird.eu/downloads/index.php).

We all know that building things from source code could be a real pain in the ass for several reasons, especially under the "low powered" Pi400 like my one ;) . So I wonder if there will be some AARCH64-packages available in the future. 

I bet a lot of users will enjoy your nice thunderbird fork like I do :).

Have a nice week everyone and thank you for your hard work!

Kind Regards,

K.

**Comment by Betterbird** [comment]
Thanks for getting in touch. The project started on Windows and due to pressure from the Linux folks we dusted off some old hardware and are compiling BB ourselves using this https://github.com/4ch1m/ansible-betterbird "one click build". So far only Arch have picked up the package. For building TB/BB/FF you need reasonably decent hardware, our Linux build takes ages (never timed it, like 1.5 hours on the vintage hardware).

Long story short: Unless someone picks it up, we're not going to provide any more binaries. We're happy to help though.

**Comment by Betterbird** [comment]
Apparently @dbonner is working on this, see https://github.com/Betterbird/thunderbird-patches/pull/48.
We won't support it officially.

**Comment by Betterbird** [comment]
@dbonner made an ARM64 build, you might want to try it:
www.betterbird.eu/downloads/LinuxArchive/betterbird-91.9.0-bb31.en-US.linux-aarch64.tar.bz2

I have the details in how that was built.
